docker/testnet/root.key

```diff
@@ -0,0 +1 @@
+ v�ୠ�=�@]j�S��(�������6�̋w
�
```

Comment:
this is regenerated on each load, we only create the file so that the contents can be overwritten within the container, this is a really "cool" docker hack.